server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,28 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class ExamEntity {
+    @Id
+    private Long id;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+    @Column(unique = true)
+    private String name;
+    private int weight;
+
+    @ManyToMany
+    @JoinColumn(name = "skill_id", referencedColumnName = "exam_id")
+    private Set<SkillEntity> skillEntities;
```

Comment:
pas utile ce joinColumn, surtout que dans ce cas, tu créer une table d'association, donc tu ne peux avoir une colonne de jointure sur cette table 
```suggestion
    @ManyToMany
    private Set<SkillEntity> skillEntities;
```